rego/lib/fugue/resource_view/terraform.rego

```diff
@@ -301,3 +306,32 @@ resource_changes_unknowns(address, prefix) = after_unknowns {
     prefixed_path = array.concat(prefix, path)
   ]
 }
+
+resource_tags(resource) = ret {
+  # AWS provider: combine `tags` and `all_tags`
+  split(resource._provider, ".")[0] == "aws"
+  tags := resource_tags_field(resource, "tags")
```

Comment:
There's a special case for AWS autoscaling groups:

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_group#tag

They can have a slightly different shape and they use the property `tag` (singular). There's also a separate resource you can use:

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_group_tag

^ That's probably out-of-scope for this code, but good to be aware of.